.github/workflows/jib-gradle-prepare-release.yml

```diff
@@ -1,38 +1,61 @@
-name: Prepare Jib Gradle release
+name: Prepare Jib release
 on:
   workflow_dispatch:
     inputs:
+      project:
+        description: Jib project to release
+        required: true
+        default: (build-plan | core | maven | gradle | extension-common | maven-extension | gradle-extension)
       release_version:
         description: new release version
         required: true
         default: (for example, 0.1.0)
 
 jobs:
   release:
-    name: Prepare Jib Gradle release
+    name: Prepare Jib release
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
         uses: actions/checkout@v2
 
-      - name: Build project
+      - name: Check input
         run: |
+          echo '* input project: "${{ github.event.inputs.project }}"'
+
+          case ${{ github.event.inputs.project }} in
+            build-plan|core|maven|gradle|extension-common|maven-extension|gradle-extension) ;;
```

Comment:
so we can't validate this in the inputs section above?

Reply:
I was not able to find any mentions about such a feature. [This article](https://levelup.gitconnected.com/how-to-manually-trigger-a-github-actions-workflow-4712542f1960) says

> Validation for these inputs is currently limited to simple presence checking, and the only input type available is a free text field (strings only).

Also found [this user question](https://github.community/t/can-workflow-dispatch-input-be-option-list/127338) asked on Aug 13 about a feature for the options-list UI.